app/src/main/java/com/spendster/presentation/homeScreen/Dashboard/ItemViewHolder.java

```diff
@@ -0,0 +1,23 @@
+package com.spendster.presentation.homeScreen.Dashboard;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.spendster.R;
+
+public class ItemViewHolder extends RecyclerView.ViewHolder{
```

Comment:
@SimonMaha what king of Item? Let's rename it to make the name describe what is the object about.